src/helm/benchmark/run_specs.py

```diff
@@ -2243,6 +2289,275 @@ def get_anthropic_hh_rlhf_spec(num_respondents: int, subset: str) -> RunSpec:
     )
 
 
+@run_spec_function("decodingtrust_stereotype_bias")
+def get_decodingtrust_stereotype_bias_spec(task: str) -> RunSpec:
+    scenario_spec = ScenarioSpec(
+        class_name="helm.benchmark.scenarios.decodingtrust_stereotype_prompts_scenario"
+                   ".DecodingTrustStereotypePromptsScenario",
+        args={},
+    )
+
+    adapter_spec = get_instruct_adapter_spec(num_outputs=25, max_tokens=150, temperature=1)
+
+    return RunSpec(
+        name=f"decodingtrust_stereotype_bias",
+        scenario_spec=scenario_spec,
+        adapter_spec=adapter_spec,
+        metric_specs=get_stereotype_bias_metric_specs(),
+        groups=["decoding_trust", "stereotype_bias"],
```

Comment:
For all the returned `RunSpec`s in this file, do:

```python
        groups=["decodingtrust", "decodingtrust_stereotype_bias"],
```

Also we should be consistent regarding `decoding_trust` versus `decodingtrust`.

Reply:
Thanks! This has been fixed in e155c21baaa7c868a7984f454b6ee68dbfd76f26.